src/KygekTeam/KygekPingTPS/Main.php

```diff
@@ -0,0 +1,64 @@
+<?php
+/*
+ * See your and others's ping and the server tps
+ * Copyright © 2021 KygekTeam
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ */
+declare(strict_types=1);
+namespace KygekTeam\KygekPingTPS;
```

Comment:
You're trying to break the plugin by changing it's namespace.

See [Poggit Rules](https://poggit.pmmp.io/rules.edit) section C1c.